modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -932,10 +932,12 @@ RETURN 'val' as two, 'val' as one
 ----
 
 Description of the returned code::
-All sub queries in a UNION should have the same ordering for the return column names, this is deprecated and will be enforced in later versions.
+All subqueries in a UNION [ALL] should have the same ordering for the return columns.
+Using differently ordered return items in a UNION [ALL] clause is deprecated and will not be possible in future versions.
```

Comment:
Maybe a nit-pick as the title of deprecations actually is ' This feature is deprecated and will be removed in future versions.'

But this sounds a bit like it covers all future versions including e.g. Neo4j 5.6, on the other hand 'and will not be possible in a future version' also sounds a bit weird. Maybe 'and will be prohibited in a future version'?

Reply:
But if we say prohibited, that would mean that it would still be possible.

Reply:
Maybe then "Using differently ordered return items in a UNION [ALL] clause is deprecated and will be removed in a future version."

Reply:
Removed sounds better to me, @Lojjs?

Reply:
Yes removed is better, in case you like to change it